### PR TITLE
Search

### PR DESCRIPTION
**Pull request by povrazor**

### CODE REVIEW

**Comment by povrazor** [comment]
Oops. I wasn't supposed to click that. Ah well.

File: public-api/vx/feed.php [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+require_once __DIR__."/../config.php";
+
+include_once __DIR__."/".CONFIG_PATH."config.php";
+require_once __DIR__."/".SHRUB_PATH."api.php";
+require_once __DIR__."/".SHRUB_PATH."node/node.php";
+
+api_Exec([
+["feed", API_GET, API_CHARGE_1, function(&$RESPONSE) {
+
+	$RESPONSE['test'] = true;
```
Comment (local-minimum) [inline public-api/vx/feed.php]: Looks like debut code left in

File: public-api/vx/search.php [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+include_once __DIR__."/".CONFIG_PATH."config.php";
+require_once __DIR__."/".SHRUB_PATH."api.php";
+require_once __DIR__."/".SHRUB_PATH."node/node.php";
+require_once __DIR__."/".SHRUB_PATH."/core/db_sphinx.php";
+
+api_Exec([
+["search", API_GET, API_CHARGE_1, function(&$RESPONSE) {
+	// q=what (like google)
+	
+	$RESPONSE['test'] = true;
```
Comment (local-minimum) [inline public-api/vx/search.php]: Idem

File: public-api/vx/feed.php (hunk 1 above)
Comment (local-minimum) [reply public-api/vx/feed.php]: *debug

Comment (povrazor) [reply public-api/vx/feed.php]: This is intended, to confirm the endpoint works.

**Pull request merged**